internal/controller/driver_controller.go

```diff
@@ -828,6 +847,7 @@ func (r *driverReconcile) reconcileNodePluginDeamonSet() error {
 										utils.PluginDirVolumeMount,
 										utils.PluginMountDirVolumeMount(kubeletDirPath),
 										utils.PodsMountDirVolumeMount(kubeletDirPath),
+										utils.If(r.logRotation(), utils.LogDirVolumeMount(r.driver.Spec.Log.Rotation.LogHostPath, daemonSet.Name), corev1.VolumeMount{}),
```

Comment:
You should not use `utils.If` and an empty volume mount definition in the volume list.
See a previous comment for the reasoning